src/providers/schema.py

```diff
@@ -18,12 +18,12 @@ class BaseProviderService:
     def __init__(self, external_address_management: bool):
         self.external_address_management = external_address_management
 
-    def list(self) -> List[Replica]:
-        """List all virtual machines in the autoscaling pool."""
+    def list(self, autoscaling: bool) -> List[Replica]:  # pylint: disable=unused-argument
```

Comment:
Without this comment, the CI fails... It must be because of the fact that this function returns something that is computed without using the parameters, whereas the other functions dont return anything